spotify_schema.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Is the spotify schema file used anywhere?
For the spotify.json.in metadata plugin file we should try to load this schema file during the build. Manually copying & escaping it should be avoided whenever possible since it's very error prone.

1. Define placeholder variable in the metadata file: `\"setup_data\": $$CFG_SCHEMA`
2. Read this schema file into the variable. Something like: `CFG_SCHEMA = "$$cat($$PWD/spotify_schema.json)"`
3. Profit! Hopefully...

Reply:
Thanks! Will update it and try!